securetea/core.py

```diff
@@ -93,6 +129,10 @@ def on_move(self, x, y):
         # Send a warning message via twitter account
         self.twitter.notify(msg)
 
+        # Send a warning message via twilio account
+        if twilio_provided:
```

Comment:
@abhisharma404 I think you should use `self.twillo_provided` at [line 44](https://github.com/OWASP/SecureTea-Project/blob/6904a604d85dc508add6fced00065de4bf792eb8/securetea/core.py#L44) for that variable to be accessible by all the class methods and also should be accessed similarly?
I think this will break the program currently.